drivers/nrf_radio_802154/platform/clock/nrf_802154_clock_zephyr.c

```diff
@@ -45,6 +45,7 @@
 
 static bool hfclk_is_running;
 static bool lfclk_is_running;
+static struct onoff_client cli;
```

Comment:
Something like `hfclk_cli` seems to be a bit more suitable name for this, as below there are function parameters and a local variable named `cli`.